### PR TITLE
jx get lang command

### PR DESCRIPTION
**Pull request by Benbentwo**
A utility option to determine what pack would be used if you were to `jx import` the current working directory

#### Examples:

<details>
    <summary>Error</summary>

```
SmithBen2289:dev benjaminsmith$ mkdir empty
SmithBen2289:dev benjaminsmith$ cd empty/
SmithBen2289:empty benjaminsmith$ /Users/benjaminsmith/dev/jx/build/jx get lang
error: failed to discover task pack in dir /Users/benjaminsmith/dev/dst/postgres-db/empty: there was an error detecting the language
SmithBen2289:empty benjaminsmith$ 
```

</details>

<details>
    <summary>Success</summary>

```
SmithBen2289:flask benjaminsmith$ ls
performing pack detection in folder /Users/benjaminsmith/dev/quickstart/flask
--> Draft detected Python (47.904483%)
selected pack: /Users/benjaminsmith/.jx/draft/packs/github.com/jenkins-x-buildpacks/jenkins-x-kubernetes/packs/python

```

</details>

#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Description
This is useful when building custom build packs or a ui for your quickstarts. you'll be able to inform the user that 
"the directory (or uploaded project etc.) has been detected to be a **XXX** type project, is this correct?" if yes -> import, if no -> add `--pack="SomeUserInput"` option

#### Special notes for the reviewer(s)
I think this can be cleaned up a little as I recycled code from different functions throughout the jx project.

### CODE REVIEW

**Comment by jstrachan** [comment]
@Benbentwo this all sounds good to me! looks like your PR needs rebasing due to a conflict

File: pkg/jx/cmd/import.go [hunk 1]
```diff
@@ -208,6 +211,41 @@ func (options *ImportOptions) Run() error {
 		return nil
 	}
 
+	if options.DetectLangOnly {
```
Comment (hferentschik) [inline pkg/jx/cmd/import.go]: I think this should really be extracted into a dedicated method. Also there seems to be a fair bit of duplicated logic in this method. I am wondering whether this somehow could be unified?

File: pkg/jx/cmd/import.go [hunk 2]
```diff
@@ -208,6 +211,41 @@ func (options *ImportOptions) Run() error {
...
+		if options.Dir == "" {
```
Comment (hferentschik) [inline pkg/jx/cmd/import.go]: This, for example, seems to be a duplicate block.

**Comment by ccojocar** [comment]
@Benbentwo Any chance to rebase this PR?

**Comment by Benbentwo** [comment]
Yes I can rebase. I changed this to WIP once I found that maven packs seem to have special logic in that section of the code base.

**Comment by Benbentwo** [comment]
It looks like this functionality has already been implemented by `jx step create task --view` but could maybe use a different flag for `--pack-only` which would stop after the output
`selected pack: ~/.jx/draft.../<pack>`

**Comment by Benbentwo** [comment]
The two methods we could use 
1. are creating a project config, and using `DiscoverBuildPack` which I believe has more verbose output, including % of packs. 
2. Using InvokeDraftPack (`similar to jx step create tasks --view`) function

**Comment by Benbentwo** [comment]
Can anyone point me in the direction of where to fix this, based on the fact its a comment commit, and looking at the source its probably the docs generation, but i'm not sure where to get started here

**Comment by abayer** [comment]
/test bdd

There's a better than even chance it's just transient noise, so I'm rerunning it.

**Pull request merged**